examples/complete-mysql/main.tf

```diff
@@ -0,0 +1,103 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+# Grab the subnets ids to be used, we are using the default VPC for the example.
+# In a real deployment you wouldn't use these subnets, not even on a Dev environment.
+data "aws_vpc" "rds" {
+  filter {
+
+    name   = "tag:Name"
+    values = ["default"]
+
+  }
+}
+
+data "aws_subnet_ids" "rds" {
+  vpc_id = data.aws_vpc.rds.id
+}
+
+# We are using the default RDS kms key for encryption.
+# Make sure you use the kms key arn
+data "aws_kms_alias" "rds" {
+  name = "alias/aws/rds"
+}
+
+# Used for demostration only, not the recommended setup for user/passwords
+resource "random_string" "rds_usr" {
+  length  = 5
+  special = false
+  upper   = false
+}
+resource "random_string" "rds_pwd" {
+  length  = 16
+  special = false
+  upper   = false
+}
+
+resource "random_string" "rds_usr2" {
+  length  = 5
+  special = false
+  upper   = false
+}
+resource "random_string" "rds_pwd2" {
+  length  = 16
+  special = false
+  upper   = false
+}
+
+# Default minimal setup
+module "rds_instance_complete" {
+  source                          = "./../../"
```

Comment:
Change source from path to url

Reply:
Hey @patrickmukumbu , since this is an example I am referencing the module we are publishing...in the root directory. The subnet group path is also supposed to change once I publish the subnet group module.

Reply:
Hey @patrickmukumbu , since this is an example I am referencing the module we are publishing...in the root directory. The subnet group path is also supposed to change once I publish the subnet group module.